app/Http/Controllers/IndexController.php

```diff
@@ -14,12 +14,12 @@
 
 class IndexController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[phpmd]__
* The method index() has 197 lines of code. Current threshold is set to 100. Avoid really long methods.

<!-- Comment by [SideCI](https://sideci.com) -->